app/views/super_admin/content.py

```diff
@@ -71,15 +72,20 @@ def placeholder_upload():
             wpercent = (basewidth / float(img.size[0]))
             hsize = int((float(img.size[1]) * float(wpercent)))
             img = img.resize((basewidth, hsize), PIL.Image.ANTIALIAS)
-            os.mkdir(app.config['TEMP_UPLOADS_FOLDER'])
+            try:
+                os.mkdir(app.config['TEMP_UPLOADS_FOLDER'])
```

Comment:
@SaptakS  the error was due to the extra `  + '/temp/') ` and without putting that try catch block, i was not able to discover `OSError: [Errno 2] No such file or directory: '/home/shubham/github/orga2/static/uploads/temp/temp/' `